compiler/testData/diagnostics/tests/multiplatform/sealedTypeAlias.kt

```diff
@@ -4,8 +4,8 @@
 // FILE: common.kt
 
 expect sealed class Presence {
-    object Online: Presence
-    object Offline: Presence
+    object Online: <!SUPERTYPE_NOT_INITIALIZED!>Presence<!>
```

Comment:
This test is failing: https://teamcity.jetbrains.com/buildConfiguration/Kotlin_KotlinForGoogle_AggregateBranch?mode=builds#3368251

It has `// FIR_IDENTICAL` and now you're making changes here, which means behavior differences. If super type is `expect`, should we bail out?

Reply:
Ah, so it looks like `isExpect` in FIR is different from `isExpect` in descriptors. The descriptor flavor actually checks if the containing class is expect and so on. So here it should be `isEffectivelyExpect`. BTW, I just realized `external` can be applied to classes for JS, so let me also use the `effective` flavor here like in FE1.0.